turbo/src/lib/i18n/zh-TW.json

```diff
@@ -233,5 +233,6 @@
     "click_to_view_subtopic": "點擊查看子主題",
     "bar": "條形",
     "circle": "圓形",
-    "chart": "圖表"
+    "chart": "圖表",
+    "these_claims_include_video_excerpts": ""
```

Comment:
@lightningorb how are you generating these translations, is it just an LLM  or google translate?

Reply:
LLM although Zoey provides us with improved translations.

Reply:
Since this is only used for Heal Michigan data right now, I'm ok leaving it as an auto-translated sentence (which I updated to match the characters we're using for "claim"). Resolved.